comps/cores/proto/api_protocol.py

```diff
@@ -160,7 +160,7 @@ class ChatCompletionRequest(BaseModel):
     logit_bias: Optional[Dict[str, float]] = None
     logprobs: Optional[bool] = False
     top_logprobs: Optional[int] = 0
-    max_tokens: Optional[int] = 1024  # use https://platform.openai.com/docs/api-reference/completions/create
```

Comment:
We need to use `max_tokens` to ensure compatibility with OpenAI.

![image](https://github.com/user-attachments/assets/f024111c-5577-43de-8f74-a254f37474a7)